lib/vanagon/platform.rb

```diff
@@ -230,7 +230,7 @@ def is_unix?
     #
     # @return [true, false] true if it is a windows variety, false otherwise
     def is_windows?
-      return !!@name.match(/^win-.*$/)
+      return !!@name.match(/^windows-.*$/)
```

Comment:
cc/ @jcoconnor and @MikaelSmith this will impact current vanagon projects that are building for windows